src/AWS.Deploy.Orchestration/CdkProjectHandler.cs

```diff
@@ -66,6 +69,13 @@ public async Task<string> ConfigureCdkProject(OrchestratorSession session, Cloud
             await using var appSettingsFile = new StreamWriter(appSettingsFilePath);
             await appSettingsFile.WriteAsync(appSettingsBody);
 
+            // The CDK bootstrap template can be generated by running 'cdk bootstrap --show-template'.
+            // We need to keep the template up to date while making sure that the 'Staging Bucket' retention policies are set to 'Delete'.
+            var cdkBootstrapTemplate = typeof(CdkProjectHandler).Assembly.ReadEmbeddedFile(TemplateIdentifier);
+            var cdkBootstrapTemplateFilePath = Path.Combine(cdkProjectPath, "CDKBootstrapTemplate.yaml");
```

Comment:
We shouldn't be writing the bootstrap template to the CDK project path. That will be really confusing for persisted deployment projects.

Reply:
Done. It is now saved to the global temp folder.